static/app/views/releases/list/releaseHealth/content.tsx

```diff
@@ -150,6 +158,14 @@ const Content = ({
                     )}
                   </AdoptionColumn>
 
+                  {adoptionStages && (
+                    <Column>
+                      {adoptionStages[project.slug]
+                        ? adoption_map[adoptionStages[project.slug]]
+                        : '---'}
```

Comment:
Not sure if this is the case, but usually we use the component `NotAvailable` when there is to be showing

Reply:
Ah okay. I'll look at that component and see what it does when I drop it in here.